lib/Dialect/AIEVec/Transforms/VectorToAIEVecConversions.cpp

```diff
@@ -71,8 +67,8 @@ extractMACOperandsFromAddOperands(Value addLhs, Value addRhs) {
     return std::make_tuple(mulOp.getLhs(), mulOp.getRhs(), acc);
 
   // If the MulIOp has been already translated to aievec::MulOp:
-  aievec::SRSOp lhsSrsOp = addLhs.getDefiningOp<aievec::SRSOp>();
-  aievec::SRSOp rhsSrsOp = addRhs.getDefiningOp<aievec::SRSOp>();
+  auto lhsSrsOp = addLhs.getDefiningOp<aievec::SRSOp>();
+  auto rhsSrsOp = addRhs.getDefiningOp<aievec::SRSOp>();
```

Comment:
Use auto when initializing with a template cast to avoid duplicating the type name